RNSound/RNSound.m

```diff
@@ -213,7 +213,9 @@ -(NSDictionary *)constantsToExport {
 }
 
 RCT_EXPORT_METHOD(play:(nonnull NSNumber*)key withCallback:(RCTResponseSenderBlock)callback) {
-  [[AVAudioSession sharedInstance] setActive:YES error:nil];
+  AVAudioSession *session = [AVAudioSession sharedInstance];
+  [session setActive:YES error:nil];
+  [session setCategory:AVAudioSessionCategoryPlayback error:nil];
```

Comment:
You don't need to do this by force, you can call `Sound.setCategory('playback')` to get the exact same effect. Here's that function: https://github.com/zmxv/react-native-sound/blob/c7e933c184fed13b857e9e29da172341cbf6fcc1/RNSound/RNSound.m#L147